db/models/score/scores.py

```diff
@@ -0,0 +1,40 @@
+"""The module containing all code related to the `scores` table
+within the Postgres db.
+"""
+import uuid
+from db import db
+from sqlalchemy import ForeignKey 
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class JustScore(db.Model):
+    """Score The sqlalchemy-flask model class used to define the
+    `scores` table in the Postgres database."""
+
+    __tablename__ = "just_scores"
```

Comment:
Can we not use abbreviations in table names? So I would prefer `scores` in this case and the class called `Score`.

Reply:
ahhh, so i called this justscores as it is what was being used in assessment frontend, can change this if score is still preferred?